includes/discovery/sensors/state/extendair.inc.php

```diff
@@ -131,6 +204,78 @@
     create_sensor_to_state_index($device, $state_name, $index);
 }
 
+$temp = snmp_get($device, 'locLinkStateV.0', "-Ovqe", "ExaltComProducts");
+$cur_oid = '.1.3.6.1.4.1.25651.1.2.4.2.3.1.15.0';
+
+if (is_numeric($temp)) {
+    $state_name = 'locLinkStateV';
+    $index = $state_name;
+    $state_index_id = create_state_index($state_name);
+
+    if ($state_index_id !== null) {
+        $states = array(
+            array($state_index_id,'almNORMAL',1,0,0) ,
```

Comment:
Wouldn't it be worth to consolidate the state translations, in this case, as they practically all are the same?